include/stl2/view/take_exactly.hpp

```diff
@@ -88,10 +93,31 @@ STL2_OPEN_NAMESPACE {
 			constexpr iter_difference_t<iterator_t<Base>> size() const noexcept { return n_; }
 			constexpr bool empty() const noexcept { return n_ == 0; }
 		};
+
+		template <InputRange R>
```

Comment:
Why `InputRange` here instead of `Range`? I think this view adaptor will work as written with output ranges.

Reply:
This one mimics `take_view`'s deduction guide. Should I update that too?

Reply:
Yes, please.

Reply:
...and I'll make a note to relax `take_view` similarly in P0896.